bin/ember-template-lint.js

```diff
@@ -115,6 +115,11 @@ function parseArgv(_argv) {
         boolean: true,
         default: false,
       },
+      printer: {
+        describe: 'Specify formatter to be used in printing output',
+        type: 'string',
+        default: 'pretty',
+      },
```

Comment:
Hmm, any reason not to go with what eslint does and call this a formatter?

Reply:
will update this to be called formatters everywhere it is called printer.

Reply:
should we rename all the things that we call printer? There is a folder in `/lib` called printers, should this be called formatters?

Reply:
> There is a folder in /lib called printers, should this be called formatters?

Ya, seems good to rename that, but maybe you could make that an "internal only" refactoring change as a separate PR (to keep the "new feature" clearly separate from the "internal renaming")?

Reply:
It's actually called `--format` in eslint. I think we should match that.